### PR TITLE
added a codebyte example/seperate from the other examples in the entry

### LINKED ISSUE #4820: [Edit] JavaScript [Edit] JavaScript: Enums
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like to edit the `Enums` concept entry under JavaScript. The entry lives in `content/javascript/concepts/enums/enums.md`.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a 'Codebyte Example' section that provides a codebyte example demonstrating the concept in use
2) Ensure that the codebyte example is different than all the other examples in the entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by TamsinLloyd99**
### Description

This pull request updates the concept entry for enums in JavaScript located in content/javascript/concepts/enums/enums.md. The update includes the addition of a new section, "Codebyte Example," which provides a practical example of how to create and use enums using Object.freeze(). The example demonstrates defining user roles, using enum values, checking permissions based on roles, and listing all possible enum values.

The new content enhances the existing documentation by offering a different context for enums, showcasing their usage in managing user permissions, and ensuring the content aligns with the concept entry template, content standards, and markdown style guide.

Motivation for this change is to provide a comprehensive and varied understanding of enums in JavaScript, giving learners practical examples they can relate to and apply in real-world scenarios.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/javascript/concepts/enums/enums.md [hunk 1]
```diff
@@ -51,3 +51,32 @@ direction: south
 direction: east
 direction: west
 ```
+
+## Codebyte Example
+
+This Codebyte example demonstrates the creation of an enum using `Object.freeze()`, how to use the enum values, and how to list all possible enum values.
```
Comment (avdhoottt) [inline content/javascript/concepts/enums/enums.md:57]:
```suggestion
This Codebyte example demonstrates the creation of an `enum` using `Object.freeze()`, how to use the enum values, and list all possible enum values:
```

File: content/javascript/concepts/enums/enums.md [hunk 2]
```diff
@@ -51,3 +51,32 @@ direction: south
...
+
+
+console.log(canEdit(UserRoles.ADMIN));  
+console.log(canEdit(UserRoles.EDITOR)); 
+console.log(canEdit(UserRoles.VIEWER)); 
+
+
+Object.keys(UserRoles).forEach((role) =>
+  console.log('role:', role)
+);
```
Comment (avdhoottt) [inline content/javascript/concepts/enums/enums.md:81]:
```suggestion
const UserRoles = Object.freeze({ 
  ADMIN: 'admin',
  EDITOR: 'editor',
  VIEWER: 'viewer'
});

let userRole = UserRoles.ADMIN;
console.log(userRole); 

function canEdit(userRole) {
  return userRole === UserRoles.ADMIN || userRole === UserRoles.EDITOR;
}

console.log(canEdit(UserRoles.ADMIN));  
console.log(canEdit(UserRoles.EDITOR)); 
console.log(canEdit(UserRoles.VIEWER)); 

Object.keys(UserRoles).forEach((role) =>
  console.log('role:', role)
);
```

**Review by avdhoottt — APPROVED** [review]
Looks good for the second review!!

**Comment by TamsinLloyd99** [comment]
Is this issue now resolved?

**Review by mamtawardhani — APPROVED** [review]
@TamsinLloyd99 thank you for your contribution, the entry looks good to be merged 😃

**Pull request merged**